examples/pytorch/graphsage_base/README.md

```diff
@@ -0,0 +1,31 @@
+Inductive Representation Learning on Large Graphs (GraphSAGE)
+============
+
+- Paper link: [http://papers.nips.cc/paper/6703-inductive-representation-learning-on-large-graphs.pdf](http://papers.nips.cc/paper/6703-inductive-representation-learning-on-large-graphs.pdf)
+- Author's code repo: [https://github.com/williamleif/graphsage-simple](https://github.com/williamleif/graphsage-simple). Note that the original code is 
+simple reference implementation of GraphSAGE.
```

Comment:
What do you mean by "simple reference"? Does the REAMDE file of `graphsage-simple` include relevant information? If so, I don't think we need to leave a note here.